Naming.md

```diff
@@ -51,7 +51,7 @@ if (checkFileSize(image))
 * activities and fragments are named with the appropriate suffix: `LoginActivity`, `RegisterFragment`
 * the corresponding layouts have to be named with the appropriate prefix: `activity_login.xml`, `fragment_register.xml`
 * all naming has to be simple, clear and mnemonic (short and meaningful)
-* the [Infinum Code Style](https://github.com/infinum/android-handbook-private/blob/master/files/InfinumCodeStyle.xml) has to be imported into Android Studio and used in formatting code. If you want to find out how to import it, check out the chapter about [Development environment setup](/books/android/Development environment setup).
+* the [Infinum Code Style](https://github.com/infinum/android-handbook-private/blob/master/files/InfinumCodeStyle.xml) has to be imported into Android Studio and used in formatting code. If you want to find out how to import it, check out the chapter about [Development environment setup](/books/android/development-environment-setup).
```

Comment:
I just realized that _Infinum Code Style_ URL does not work if you are not logged in as that resource is on our private repository. I'm guessing the only solution is to move it to the public repository. I'll see if we want this to be public or not.